nodejs/eks/cluster.ts

```diff
@@ -74,13 +74,18 @@ export interface CoreData {
     subnetIds: pulumi.Output<string[]>;
     clusterSecurityGroup: aws.ec2.SecurityGroup;
     provider: k8s.Provider;
-    instanceRole: pulumi.Output<aws.iam.Role>;
     instanceProfile: aws.iam.InstanceProfile;
     eksNodeAccess?: k8s.core.v1.ConfigMap;
     kubeconfig?: pulumi.Output<any>;
     vpcCni?: VpcCni;
 }
 
+const eksManagedPolicyArns: string[] = [
```

Comment:
Nit - but perhaps calling out (in the variable name) that these are the "instance" role policies might make sense.  There are two roles involved, and the sets of policies for each are different.